readme.md

```diff
@@ -14,8 +14,8 @@
 | 🐍 **Python**     | Lettuce                     | `lettuce-python-sample`        | ⏳ Pending   |                                                                         |                                                                                                                                                           |
 | 🐍 **Python**     | unittest                    | `unittest-python-sample`       | ⏳ Pending   |                                                                         |                                                                                                                                                           |
 | 🟦 **TypeScript** | Playwright                  | `playwright-typescript-sample` | ✅ Completed |                                                                         |                                                                                                                                                           |
-| 🟦 **TypeScript** | Cypress                     | `cypress-typescript-sample`    | ⏳ Pending   |                                                                         |
-| 🟦 **TypeScript** | Jest                        | `jest-typescript-sample`       | ⏳ Pending   |                                                                         |                                                                                                                                                           |
+| 🟦 **TypeScript** | Cypress                     | `cypress-typescript-sample`    |  ✅ Completed |  [🔗](https://github.com/srthorat/sauce-lab-ats-cypress-typescript )   | [Sakharam Thorat](https://github.com/srthorat)
```

Comment:
Please follow the existing format for the contributor using the profile image. thank you